bin.ts

```diff
@@ -148,7 +148,9 @@ function getRemoteUrl(https = true) {
       remoteUrl.protocol = "https:";
     }
 
-    return remoteUrl.href;
+    // remove trailing slashes
+    var sanitizedUrl = remoteUrl.href.replace(/\/+$/, "")
+    return sanitizedUrl;
```

Comment:
please, use `const` or remove the line and return the result with `return remoteUrl.href.replace(/\/+$/, "")`